### DISCUSSION TITLE
blog/posts/230311-airbridge-react-native-webview/

### DISCUSSION THREAD

**Discussion by giscus** [ricale/blog#125] [Giscus]
# blog/posts/230311-airbridge-react-native-webview/

에어브릿지Airbridge 는 모바일 앱의 마케팅 성과 분석을 위한 솔루션이다. 앱 안에 SDK 를 삽입하면 사용자가 어떤 광고를 통해 앱에 들어왔으며 어떤 액션까지 도달했는지를 (요컨대 설치만 했는지, 가입까지 했는지, 결제까지 했는지) 추적해준다.…

https://ricale.kr/blog/posts/230311-airbridge-react-native-webview/

**Comment by Lee-sungheon** [comment]
react native webview에 native한 외부 sdk를 심는 경우에 많은 도움이 되었습니다~!
귀중한 포스팅 감사합니다 🙇‍♂️
